test/test_nn.py

```diff
@@ -3480,6 +3488,7 @@ def test_multihead_attn_all_arguments3():
         with self.assertRaisesRegex(AssertionError, "bias cannot be added to static key."):
             test_multihead_attn_all_arguments2()  # Test MultiheadAttention with all the argument.
         test_multihead_attn_all_arguments3()  # Test MultiheadAttention with all the argument.
+        test_multihead_attn_all_arguments4()  # Test MultiheadAttention with all the argument.
```

Comment:
All three functions here have the same comment after them (test_multihead_attn_all_arguments4/3/2). How come?

Reply:
The four functions `test_multihead_attn_all_arguments4/3/2` set up `_multihead_attn_test_helper` func differently.